warp10/src/main/java/io/warp10/script/ext/token/TokenWarpScriptExtension.java

```diff
@@ -16,57 +16,61 @@
 
 package io.warp10.script.ext.token;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Properties;
-
 import io.warp10.WarpConfig;
-import io.warp10.WarpDist;
 import io.warp10.crypto.KeyStore;
 import io.warp10.standalone.Warp;
 import io.warp10.warp.sdk.WarpScriptExtension;
 
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
 public class TokenWarpScriptExtension extends WarpScriptExtension {
-  
+
   /*
-   *  Name of configuration key with the token secret. 
+   *  Name of configuration key with the token secret.
+   *
+   * Must be warp.key.token.secret if handled by OSS
    */
   public static final String CONF_TOKEN_SECRET = "token.secret";
-  
+
   /**
    * Current Token Secret
    */
   public static String TOKEN_SECRET = null;
-  
-  private static final Map<String,Object> functions = new HashMap<String,Object>();
+
+  private static final Map<String, Object> functions = new HashMap<String, Object>();
   private static final KeyStore keystore;
-  
+
   static {
     TOKEN_SECRET = WarpConfig.getProperty(CONF_TOKEN_SECRET);
```

Comment:
Use separate constants to make the code clearer, even though they have the same value. CONF_TOKEN_SECRET for the configuration key, KEY_TOKEN_SECRET for the KeyStore key. Clearly describe the behavior in the comments. As it is now it is rather hard to understand since you say "Myst be warp.key.token.secret is handled by OSS" with a constant defined below. What should I do if I want to use OSS? Recompile the code with a different value of CONF_TOKEN_SECRET?